### PR TITLE
Vox nerfs the big umpf

### PR DESCRIPTION
**Pull request by Rayfod**
Vox nerfs: Kosmag removed, Armalis changed to one, and the sniper rifle nerfed so that it **cannot** stun anymore while giving it back its original ammo.
Nerfed the dark matter gun, to charge a focused shot, every 12 seconds.
These changes were talked about with AntonKR before they were made.
Final confirmation from Carl would be good.
Let's see how this goes.

### CODE REVIEW

**Comment by Anton-Kr** [comment]
Ill ask for Carl’s approval prior to merging this.

File: maps/away/voxship/voxship_jobs.dm [hunk 1]
```diff
@@ -210,8 +210,8 @@
 /obj/item/weapon/gun/energy/sniperrifle/vox
```
Comment (BurpleBineapple) [inline maps/away/voxship/voxship_jobs.dm:210]: Hot take, but I'd say remove this entirely. A self-recharging practically infinite ammo 1 shot knockout gun is pretty meme.

Comment (Rayfod) [reply maps/away/voxship/voxship_jobs.dm:210]: I changed the stun instead of removing the gun.

**Comment by Rayfod** [comment]
#1349 are going to take my nerfs into his PR.

**Pull request closed without merging**